client/src/components/ReviewModal/CreateReviewModal/CreateReviewModal.js

```diff
@@ -33,6 +33,9 @@ export default class CreateReviewModal extends React.Component {
             case DateFormat.EXACT_DATE:
                 date = prettyPrintDate(form.date, form.dateSelectValue);
                 break;
+            case DateFormat.UNKNOWN:
```

Comment:
Seems like there is some weirdness with the updateReviewModal

-> Uninstalled date unknown is coming up as NULL change
-> changing from date unknown to a known date is not coming up as a change at all

Reply:
fixed